ovs-persistence/src/main/java/org/dcsa/ovs/persistence/entity/LocationEntity.java

```diff
@@ -0,0 +1,42 @@
+package org.dcsa.ovs.persistence.entity;
+
+import lombok.*;
+import org.dcsa.skernel.domain.persistence.entity.Address;
+import org.dcsa.skernel.domain.persistence.entity.Facility;
+
+import java.util.UUID;
+import javax.persistence.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PUBLIC)
+@Entity
+@Table(name = "location")
+public class LocationEntity {
```

Comment:
It should be `Location` rather than `LocationEntity`, I do realize this would cause a conflict with the existing entity file, here's how you could ignore it using `excludeFilters`

```
@Configuration 
@ComponentScan(basePackages = {"com.example"}, excludeFilters={
@ComponentScan.Filter(type=FilterType.ASSIGNABLE_TYPE, value=Foo.class)})
public class MySpringConfiguration {}
```